src/components/Join us/JoinUs.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import Link from "next/link";
+import Carousel from 'react-bootstrap/Carousel'; 
+
+
+
+
+const JoinUs: React.FC = () => {
+  return (
+    <>
+        <div className="w-2060 h-1288 grid grid-cols-2 gap-10 content-start">
```

Comment:
In general, for this component, you use a lot of manual and hard-coded margin values. Please use `flex-box` instead. The general approach I would go about this is that I'd group the header (e.g, "Join our team") and description (e.g., "Get involved with out team") into a item (a.k.a, div) and the image as another item. So that when I apply `flex flex-row` (tailwindcss) to the outter most component, I'd see text on the left and image on the right.